pkgs/desktops/plasma-5/default.nix

```diff
@@ -83,7 +82,6 @@ let
             setupHook = args.setupHook or defaultSetupHook;
 
             meta = {
-              broken = lib.versionAtLeast qtbase.version "5.15";
```

Comment:
Plasma 5.18 does not support Qt 5.15.

Reply:
Why?

Reply:
because it's based on qt 5.12 and 5.13 https://community.kde.org/Schedules/Plasma_5